_sass/sass/components/nav.sass

```diff
@@ -73,6 +73,7 @@ a.nav-item:not(.button)
     flex-basis: 0
 
 .nav-left
+  overflow: hidden
```

Comment:
Should be listed in alphabetical order.

Reply:
@kevva I don't really order alphabetically. When I did CSS a lot before, I used to order layout type props first, and then visual only. I don't see any benefit in doing it alphabetically tbh.